src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const amountRent = 40;
+
+  if (days < 3) {
+    return amountRent * days;
+  }
+
+  if (days >= 3 && days < 7) {
```

Comment:
what does these numbers mean?
Don't use literals in expressions (HARDCODE). Instead use constants explaining this values

![image](https://github.com/mate-academy/js_task-transportation-on-vacation/assets/91946036/4de27dbe-3b93-4ced-b0e5-b8c7e92287ee)

[CHECKLIST](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md)